src/main/java/AddTasks.java

```diff
@@ -0,0 +1,75 @@
+
+public class AddTasks {
+
+
+    private int i;
```

Comment:
i is usually used as a temporary variable, such as in the for loop, etc. Thus here I think maybe giving a specific name will be better and it will improve the readability.